### PR TITLE
Add new citations of Catalyst / PUDL

### PR DESCRIPTION
**Pull request by zaneselvans**
# Overview

I was searching Google Scholar for other things, and came across some additional references to PUDL / Catalyst, so I thought I would add them to the bibliography.

# Testing

I built the docs locally and viewed the Bibliography page.

```[tasklist]
# To-do list
- [ ] Ensure docs build, unit & integration tests, and test coverage pass locally with `make pytest-coverage` (otherwise the merge queue may reject your PR)
- [ ] For significant ETL changes, ensure the full ETL runs locally
- [ ] For major data coverage & analysis changes, [run data validation tests](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation)
- [ ] If updating analyses or data processing functions: make sure to update or write data validation tests
- [ ] Update the [release notes](../docs/release_notes.rst): reference the PR and related issues.
- [ ] Review the PR yourself and call out any questions or issues you have
```

### CODE REVIEW

File: docs/catalyst_cites.bib [hunk 1]
```diff
@@ -214,3 +214,113 @@ @article{doi:10.1146/annurev-environ-020220-061831
             we identify gaps in this literature, propose future research directions, and
             discuss important considerations for deployment.}
 }
+
+@misc{zheng2024optimal,
+      title={Optimal transmission expansion minimally reduces decarbonization costs of U.S. electricity},
```
Comment (e-belfer) [inline docs/catalyst_cites.bib:219]: This is rendering as "u.s. electricity" for me for some reason.

File: docs/catalyst_cites.bib [hunk 2]
```diff
@@ -214,3 +214,113 @@ @article{doi:10.1146/annurev-environ-020220-061831
...
+      decreases the cost of a 100% clean system by only 4% compared to a plan that
+      relies solely on current transmission. Expanding capacity only within existing
+      interconnects can achieve most of these savings. Adjustments to energy storage and
+      generation mix can leverage the current interregional transmission infrastructure
+      to build a clean power system at a reasonable cost.}
+}
+
+@article{10.1257/jep.37.4.155,
+      Author = {Davis, Lucas W. and Hausman, Catherine and Rose, Nancy L.},
+      Title = {Transmission Impossible? Prospects for Decarbonizing the US Grid},
```
Comment (e-belfer) [inline docs/catalyst_cites.bib:245]: US also renders as "us" for me

**Review by e-belfer — CHANGES_REQUESTED** [review]
There are weird lower-case typos, and they're not the only one on the bib page (see august 2021, e.g.). Seems like something weird is happening with capitalization during the rendering process.

If you'd prefer to take that up in a separate issue, this otherwise all looks ok to me. Links all work as expected, etc.

**Comment by zaneselvans** [comment]
The capitalization is BibTeX/LaTeX doing its thing and applying naive title case to everything. I could fix it with liberal use of curlybraces, but I'm just cut-and-pasting the BibTeX citations from journals so that we have a reference library of some kind to look at later.

**Comment by bendnorman** [comment]
Would it be possible to store PUDL citations in a stand-alone structured file to easily convert the information to other formats? For example, our Sloan interim report wants a table of PUDL citations.

**Comment by zaneselvans** [comment]
I went ahead and fixed the capitalization issues I saw in the new records. We should probably do an annual lit review at least and make sure everything is clean and up to date...

File: docs/further_reading.bib [hunk 3]
```diff
@@ -278,3 +300,57 @@ @article{HIRTH2020100433
...
+    techniques, we recognize that data cleaning is still an active field of research and
+    therefore provide future directions of research whenever appropriate.}
+}
+
+@book{doi:https://doi.org/10.1002/0470036427,
+publisher = {John Wiley & Sons, Ltd},
+isbn = {9780470036426},
+author = {von Meier, Alexandra},
+title = {Electric Power Systems: A Conceptual Introduction},
+doi = {https://doi.org/10.1002/0470036427},
```
Comment (e-belfer) [inline docs/further_reading.bib:342]: Should this just be `10.1002/0470036427` like the other ones?

Comment (zaneselvans) [reply docs/further_reading.bib:342]: Bah yes.

**Review by e-belfer — APPROVED** [review]
Tiny nits remaining. Will approve this so you can fix them and hit merge, though!

**Pull request merged**